lib/backend.ml

```diff
@@ -274,7 +274,12 @@ module Performance = struct
     in
     let resp =
       (* TODO: make a cli-argument out of this instead of doing this always *)
-      let responses = List.map Merlin.Response.crop_value responses in
+      let responses =
+        List.map
+          (fun resp ->
+            Merlin.Response.(crop_value @@ crop_heap_and_cache @@ resp))
+          responses
+      in
```

Comment:
When analyzing performance, we don't want to crop `mbytes`. It's a very important number for analyzing memory performance. I also wouldn't crop the info on cache hits and misses, as it's interesting for performance analysis as well. 

For context on why we get rid of `value`: It's because for certain queries it's huge, so it makes the json's and write time to them enormous.